python/ray/actor.py

```diff
@@ -1768,7 +1766,9 @@ def exit_actor():
     This API can be used only inside an actor. Use ray.kill
     API if you'd like to kill an actor using actor handle.
 
-    When the API is called, the actor raises an exception and exits.
+    When the API is called, the actor sets a flag to exit.
+    The worker periodically checks the flag and exits the actor.
+    So there may be some delay before the actor actually exits.
```

Comment:
For non-asyncio actors, we should still be raising the exception immediately, right? Don't want a behavior change there